doc/Language/grammar_tutorial.pod6

```diff
@@ -27,41 +27,75 @@ language.
 
 =SUBTITLE The Broad Concept of Perl Grammars
 
-Regular Expressions (regex) work well to find patterns in strings and manipulate them. However, when you need to find multiple patterns at once, or need to combine patterns, or test for patterns that may surround strings, or other patterns - regular expressions alone are not adequate.
```

Comment:
Sounds a bit unstable for me, but the initial intention of changes is good. What do you think about improving it with an automatic line-wrapper?
And maybe it will be good to write a test for too long lines... It can be written easily as a checker, and we have Text::Fill in our ecosystem to even fix that automatically in Perl 6. I'll ask tomorrow in #perl6 channel about how important it is.